rfcs/0012-gossip.md

```diff
@@ -10,34 +10,30 @@ Gossip runs on a well-known UDP/IP port, or a port in a well-known range.  Once
 
 ## Gossip Records
 
-Records shared over gossip are arbitrary, but signed and versioned (with a timestamp), and need to make sense to the node receiving them.  A strategy of "most recently recevied wins" is employed to manage what's stored in cluster info.
+Records shared over gossip are arbitrary, but signed and versioned (with a timestamp), and need to make sense to the node receiving them.  A strategy of "most recent timestamp wins" is employed to manage what's stored in cluster info.  If 2 values from the same creator have the same timestamp, the value with the greater hash wins.
 
 ## Push
 
 ### Push Message
 
-A push message is "hey network, I have something new to share".  Nodes send push messages to a `fanout` number of push peers.
+A push message is "hey network, I have something new to share".  Nodes send push messages to `PUSH_FANOUT` push peers.
 
 Upon receiving a push message, a node examines the message for:
 
-1. duplication: if the message has been seen before, the node responds with a "prune" message drops the message.
+1. duplication: if the message has been seen before, the node responds with `PushMessagePrune` and drops the message
```

Comment:
i would also add another statement here.  IF the signed timestamp is too old or to new the message is dropped.  This ensures that the message `push` is short lived.  Once `push` is done, the pull protocol propagates these messages lazily throughout the network.